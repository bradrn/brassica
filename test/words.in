baalaq
bin
bisa
buthe
dafin
danit
daqat
simi
dawa
ŋuwe
debu
din
dise
fas
fasi
fawetl
fawetli
feqe
fetlhalh
gan
gadiq
gadye
galab
ganit
welus
gatlaq
nge
haŋuhaŋu
heŋheŋ
kaq
kasip
khaliŋ
phula
khambuy
laqit
yenge
khenan
khunu
kube
kulit
saqam
kumay
kuwi
lampala
laqit
khambuy
mebas
layat
lheŋe
lhibuq
lhiise
lhiisi
lis
mandaŋ
mandaŋ
maqu
matsiŋ
may
mebas
yenge
laqit
meŋul
naŋa
nawun
ndeqis
ndisi
ndiyam
ngiya
ŋalep
ŋandu
ŋandi
ŋuwe
dawa
pan
paatli
paatli
phage
phaŋi
phingis
qangat
phula
khaliŋ
phuum
qaabew
qaaŋ
qaathan
qaathan
qalit
nqam
nam
qangat
qanta
qanu
qaŋeth
qefaa
qefay
qetlaag
qisa
qisni
qitsim
qumukh
qumum
qume
saalhe
safilh
sakhul
sal
saqam
kulit
sasay
sasuf
segaag
segek
segeg
siifa
siki
simi
daqat
siwe
siwi
tasek
thaba
thabu
thalaŋ
tham
thani
tlaame
tlaquf
lalef
tluŋse
tsagif
tsakaq
tsektsek
tsisa
walha
waalhi
wanaŋ
waq
mah
wayug
wedeŋ
wediy
welus
ganit
wiilets
witsi
yaaq
yagi
yagi
yangat
yaqa
nye
yenge
mebas
yusaa
yusaye

[syllabification stress test]
wakatuukatuu watsikatuukatuu wakekatuukatuu waŋukatuukatuu walikatuukatuu
mahkatuukatuu mahtsikatuukatuu mahkekatuukatuu mahŋukatuukatuu mahmukatuukatuu

[word/clitic boundary tests]
siw=dafin si=dafin siwedafin siwdafin sidafin 
tse=dafin tsedafin ŋa=dafin ŋadafin

siw=fawetli si=fawetli siwefawetli siwfawetli sifawetli 
tse=fawetli tsefawetli ŋa=fawetli ŋafawetli

siw=khaliŋ si=khaliŋ siwekhaliŋ siwkhaliŋ sikhaliŋ 
tse=khaliŋ tsekhaliŋ ŋa=khaliŋ ŋakhaliŋ

siw=saqam si=saqam siwesaqam siwsaqam sisaqam 
tse=saqam tsesaqam ŋa=saqam ŋasaqam

siw=thalaŋ=baalaq si=thalaŋ=baalaq siwethalaŋ=baalaq siwthalaŋ=baalaq sithalaŋ=baalaq 
tse=thalaŋ=baalaq tsethalaŋ=baalaq ŋa=thalaŋ=baalaq ŋathalaŋ=baalaq