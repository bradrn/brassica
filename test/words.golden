SCA Output: banaʔ
SCA Output: bin
SCA Output: bes
SCA Output: bus
SCA Output: dfin
SCA Output: dnir
SCA Output: daʔr
SCA Output: sim
SCA Output: daw
SCA Output: uŋu
SCA Output: dob
SCA Output: gin
SCA Output: gis res
SCA Output: fas
SCA Output: far
SCA Output: fwas fwes
SCA Output: fwes
SCA Output: feʔ
SCA Output: flal
SCA Output: an
SCA Output: igiʔ
SCA Output: agi egi
SCA Output: inab
SCA Output: inir
SCA Output: wnus
SCA Output: isaʔ
SCA Output: g
SCA Output: oŋoŋ
SCA Output: aŋaŋ aŋeŋ eŋaŋ eŋeŋ
SCA Output: aʔ
SCA Output: isip
SCA Output: kniŋ
SCA Output: fon
SCA Output: kbuy
SCA Output: niʔr
SCA Output: yeg
SCA Output: knan
SCA Output: kun
SCA Output: ub
SCA Output: inir
SCA Output: saʔm
SCA Output: imay
SCA Output: u iw
SCA Output: nampan
SCA Output: niʔr
SCA Output: kbuy
SCA Output: mbas
SCA Output: nyar
SCA Output: leŋ
SCA Output: lbuʔ
SCA Output: lis les
SCA Output: lir
SCA Output: nis
SCA Output: ndo
SCA Output: ndo
SCA Output: moʔ
SCA Output: msiŋ
SCA Output: may
SCA Output: mbas
SCA Output: yeg
SCA Output: niʔr
SCA Output: mŋun
SCA Output: naŋ
SCA Output: nwun
SCA Output: ndiʔs
SCA Output: ndir
SCA Output: ndyam
SCA Output: eŋgey
SCA Output: aŋnap eŋnep
SCA Output: oŋod
SCA Output: eŋed
SCA Output: uŋu
SCA Output: daw
SCA Output: pan
SCA Output: pes
SCA Output: pes
SCA Output: fa fe
SCA Output: feŋ
SCA Output: fgis
SCA Output: aŋgar
SCA Output: fon
SCA Output: kniŋ
SCA Output: fum
SCA Output: abaw abew
SCA Output: aŋ
SCA Output: asan
SCA Output: asan
SCA Output: nir
SCA Output: naʔm
SCA Output: nam
SCA Output: aŋgar
SCA Output: an at
SCA Output: on
SCA Output: aŋas eŋes
SCA Output: fa
SCA Output: fay
SCA Output: ta
SCA Output: es
SCA Output: is in
SCA Output: tim
SCA Output: muk
SCA Output: mum
SCA Output: um
SCA Output: sal sel
SCA Output: sfil
SCA Output: skun
SCA Output: san
SCA Output: saʔm
SCA Output: inir
SCA Output: say
SCA Output: suf
SCA Output: sa
SCA Output: sa se
SCA Output: sa se
SCA Output: sef
SCA Output: si
SCA Output: sim
SCA Output: daʔr
SCA Output: siw sew
SCA Output: siw
SCA Output: tsa tse
SCA Output: tab
SCA Output: tob
SCA Output: tno
SCA Output: tam
SCA Output: ten
SCA Output: tam tem
SCA Output: tuʔf
SCA Output: naf nef
SCA Output: tuŋ tus
SCA Output: tif
SCA Output: taʔ
SCA Output: tasa tase tesa tese
SCA Output: tes
SCA Output: wal
SCA Output: wel
SCA Output: wno
SCA Output: waʔ
SCA Output: ma
SCA Output: wyu
SCA Output: wraŋ wreŋ
SCA Output: wgi
SCA Output: wnus
SCA Output: inir
SCA Output: winas wines
SCA Output: wis
SCA Output: yaʔ
SCA Output: ye
SCA Output: ye
SCA Output: ygar
SCA Output: yaʔ
SCA Output: ni
SCA Output: yeg
SCA Output: mbas
SCA Output: ysa
SCA Output: ysay ysey
SCA Output: 
SCA Output: [syllabification stress test]
SCA Output: waroru wseroru waroru wŋororu wneroru
SCA Output: maroru maseroru maaroru maŋororu mamororu
SCA Output: 
SCA Output: [word/clitic boundary tests]
SCA Output: siwdfin rdfin rwarfin sewdfin serfin 
SCA Output: tdfin tarfin ŋdfin aŋarfin
SCA Output: 
SCA Output: siwfwes sfwes rwafwes sewfwes sefwes 
SCA Output: tfwes tafwes ŋfwes aŋafwes
SCA Output: 
SCA Output: siwkniŋ skniŋ rwakniŋ sewkniŋ sekniŋ 
SCA Output: tkniŋ takniŋ ŋkniŋ aŋakniŋ
SCA Output: 
SCA Output: siwsaʔm ssaʔm rwasaʔm sewsaʔm sesaʔm 
SCA Output: tsaʔm tasaʔm aŋsaʔm aŋasaʔm
SCA Output: 
SCA Output: siwsnobanaʔ ssnobanaʔ rwasnobanaʔ sewsnobanaʔ sesnobanaʔ 
SCA Output: tnobanaʔ tasnobanaʔ ŋsnobanaʔ aŋasnobanaʔ
